pyproject.toml

```
@@ -6,11 +6,15 @@ authors = [
 ]
 
 dependencies = [
+    "numpy",
+    "ase==3.22.1",
+    "metatensor[torch]",
+    "rascaline-torch@git+https://github.com/luthaf/rascaline#subdirectory=python/rascaline-torch",
```

you may want to pin rascaline to a specific version, for example for the latest commit
```suggestion
    "rascaline-torch@git+https://github.com/luthaf/rascaline@e215461#subdirectory=python/rascaline-torch",
```